2-mandatory/1-weather-report.js

```diff
@@ -12,7 +12,12 @@
 */
 
 function getTemperatureReport(cities) {
-    // TODO
+      let  temperature=[];
+    // for(let i = 0;i < cities.length; i++){
+        for(const city of cities){
+        temperature.push(`The temperature in ${city} is ${temperatureService(city)} degrees`)
+    }
+    return temperature;
```

Comment:
when you have an array that needs to be transformed, input and output have the same size, you can also use `map`

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map